extensions/objects/templates/math-equation-editor.component.spec.ts

```diff
@@ -65,14 +65,19 @@ describe('MathEquationEditor', () => {
       MathEquationEditorComponent);
     component = fixture.componentInstance;
     window.Guppy = MockGuppy;
-    component.currentValue = '';
   });
 
   afterEach(() => {
+    // This throws "The operand of a 'delete' operator must be optional".
+    // We need to suppress this error because "Property Guppy is not an
+    // optional property in global interface Window. A property needs to
+    // be optional or have undefined as a union type in order to be deleted".
+    // @ts-ignore
     delete window.Guppy;
   });
 
   it('should add the change handler to guppy', () => {
+    component.currentValue = '';
```

Comment:
Why this needs to be added?

Reply:
We need to test this functionality `'should not show warnings if the editor is active'` (just below) so previously `component.currentValue` was assigned an `undefined` value. But we can't do this because we''ll have to introduce undefined as a type. So I removed this statement from beforeEach and added it in individual `it` statements such that in particular `it` statement mentioned above it remains undefined.